x-pack/plugins/grokdebugger/public/plugin.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { first } from 'rxjs/operators';
+import { registerFeature } from './register_feature';
+import { PLUGIN } from '../common/constants';
+
+const inactiveLicenseMessage = i18n.translate('xpack.grokDebugger.clientInactiveLicenseError', {
+  defaultMessage: 'The Grok Debugger tool requires an active license.',
+});
+
+export class Plugin {
+  setup(coreSetup, plugins) {
+    registerFeature(plugins.home);
+
+    this.licensing = plugins.licensing;
+
+    this.licensing.license$.pipe(first()).subscribe(license => {
+      plugins.devTools.register({
+        order: 6,
+        title: i18n.translate('xpack.grokDebugger.displayName', {
+          defaultMessage: 'Grok Debugger',
+        }),
+        id: PLUGIN.ID,
+        enableRouting: false,
+        disabled: !license.isActive,
+        tooltipContent: !license.isActive ? inactiveLicenseMessage : null,
+        async mount(context, { element }) {
+          const [coreStart] = await coreSetup.getStartServices();
+          const { renderApp } = await import('./render_app');
+          return renderApp(element, coreStart);
+        },
+      });
+    });
+  }
+
+  start(coreStart) {
+    this.licensing.license$.subscribe(license => {
+      coreStart.chrome.navLinks.update(PLUGIN.ID, {
+        hidden: !license.isActive,
```

Comment:
It's because the subscription in `start` will be overriding your hardcoded `true` back to `false`. If you also comment out the `start` logic it produces this:

![Screenshot 2020-03-30 13 11 25](https://user-images.githubusercontent.com/471693/77910995-fc768a80-7287-11ea-8159-643991fc4885.png)

Reply:
(Nonetheless this logic will change given the other feedback from @jloleysens)